src/peep/peep.c

```diff
@@ -8401,10 +8437,10 @@ static bool peep_should_use_cash_machine(rct_peep *peep, int rideIndex)
  */
 static void sub_69A98C(rct_peep *peep)
```

Comment:
'reset_pathfind_goal' isn't really what this does. I'm thinking `pathfind_choose_direction` would be more descriptive.